SpatialGDK/Source/SpatialGDKEditorToolbar/Public/SpatialGDKEditorToolbar.h

```diff
@@ -81,6 +97,21 @@ class FSpatialGDKEditorToolbarModule : public IModuleInterface, public FTickable
 	bool StopSpatialServiceIsVisible() const;
 	bool StopSpatialServiceCanExecute() const;
 
+	void OnToggleSpatialNetworking();
+	bool OnIsSpatialNetworkingEnabled() const;
+
+	void GDKEditorSettingsClicked() const;
```

Comment:
This is a bit of a cluster, is it in the same order as the cpp methods and perhaps we can get some whitespace?